scripts/run-for-subtyping.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/sh
+
+set -e
+set -o pipefail
+
+printf "Start generating pre-release files...\n\n"
+
+# Set locations for s3 bucket that contains subtyping commit files
+URL="s3://d3b-openaccess-us-east-1-prd-pbta/open-targets"
+COMMIT="analysis-files-pre-release-commit"
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# If RUN_LOCAL is used, the time-intensive steps are skipped because they cannot
+# be run on a local computer -- the idea is that setting RUN_LOCAL=1 will allow for
+# local testing running/testing of all other steps
+RUN_LOCAL=${RUN_LOCAL:-0}
+
+# Get base directory of project
+cd ..
+BASEDIR="$(pwd)"
+cd -
+  
+analyses_dir="$BASEDIR/analyses"
+data_dir="$BASEDIR/data"
+scratch_dir="$BASEDIR/scratch"
+
+# Compile all the files that need to be examined for changes in one place
+# in the scratch directory
+commit_dir="${scratch_dir}/analysis-files-pre-release-commit"
+mkdir -p ${commit_dir}
+
+# Create the independent sample list using the *base* histology file (i.e. - histologies-base.tsv)
+echo "Create independent sample list for fusion filtering module"
+cd ${analyses_dir}/independent-samples
+OPENPBTA_BASE_SUBTYPING=1 bash run-independent-samples.sh
+
+
+## Step 1. Generate summary files needed for subtyping
+
+# Run GSEA
+echo "Run GSEA"
+cd ${analyses_dir}/gene-set-enrichment-analysis
+OPENPBTA_BASE_SUBTYPING=1 bash run-gsea.sh
+
+# Copy over GSEA results for subtyping
+cp ${analyses_dir}/gene-set-enrichment-analysis/results/gsva_scores.tsv ${commit_dir}
+
+# Run TP53
+echo "TP53 altered score"
+cd ${analyses_dir}/tp53_nf1_score
+OPENPBTA_BASE_SUBTYPING=1 bash run_classifier.sh
+
+# Copy over TP53 results
+cp ${analyses_dir}/tp53_nf1_score/results/* ${commit_dir}
+cp ${analyses_dir}/tp53_nf1_score/plots/* ${commit_dir}
```

Comment:
@ewafula thanks for updating this - I think these can remain in the generate analysis files script since they are not subtyping modules.

Reply:
moved gsea and tp53 to generate analysis files script